dask_deltatable/core.py

```diff
@@ -117,11 +115,14 @@ def _read_from_filesystem(
         meta = meta[columns]
 
     return dd.from_map(
-        partial(_read_delta_partition, fs=fs, columns=columns, schema=schema, **kwargs),
+        _read_delta_partition,
         pq_files,
         meta=meta,
         label="read-delta-table",
-        token=tokenize(path, fs_token, **kwargs),
+        fs=fs,
+        columns=columns,
+        schema=schema,
+        **kwargs,
```

Comment:
`from_map` in `dask-expr` doesn't support `token=`, but in this case I think we can have `from_map` handle tokenization internally (someone should feel free to correct me if that's not the case)